.github/workflows/yocto-build.yml

```diff
@@ -63,13 +71,27 @@ on:
         required: false
         type: boolean
         default: false
-    outputs:  
-      device_slug: 
+      QEMU_CPUS:
+        required: false
+        type: number
+        default: 4
+      QEMU_MEMORY:
+        type: string
+        required: false
+        default: "1G"
```

Comment:
Let's hardcode these values in the workflow where we call the action. Only 2-3 repositories use QEMU for testing anyway so these fields are useless most of the time.